pootle/apps/pootle_app/management/commands/clear_stats.py

```diff
@@ -12,11 +12,45 @@
 # This must be run before importing Django.
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
 
-from pootle_app.management.commands import PootleCommand
+from optparse import make_option
 
+from django.core.management.base import NoArgsCommand
 
-class Command(PootleCommand):
-    help = "Flush stats cache."
+from django_redis import get_redis_connection
+from redis import WatchError
 
-    def handle_all_stores(self, translation_project, **options):
-        translation_project.clear_all_cache()
+from pootle.core.mixins.treeitem import POOTLE_DIRTY_TREEITEMS
+from pootle.core.models import Revision
+
+
+class Command(NoArgsCommand):
+    help = """Flush stats cache and dirty tree item counter and
+    rq data (optional).
+    """
+    shared_option_list = (
+        make_option(
+            "--clear-rq",
+            action="store_true",
+            default=False,
+            help=(u"Clear all RQ data"),
+        ),
+    )
+    option_list = NoArgsCommand.option_list + shared_option_list
```

Comment:
Nit: There is no need to add a unnecessary `shared_option_list` variable. See https://github.com/translate/pootle/blob/master/pootle/apps/pootle_app/management/commands/initdb.py#L24-L32